src/pages/ListView/ListView.js

```diff
@@ -11,19 +14,32 @@ const ListView = ({ listId, handleModalOpen, token }) => {
     db.collection(`lists/${listId}/items`),
   );
 
+  const [showDetails, setShowDetails] = useState(false);
+
+  const toggleDetailView = () => {
+    setShowDetails(!showDetails);
+  };
+
```

Comment:
Here is where I've moved `showDetails` from ShoppingListItem.js to ListView. and I'm now passing it down to ListHeader and ShoppingList -> ShoppingListItem